.dotnet/api/OpenAI.netstandard2.0.cs

```diff
@@ -1146,14 +1146,24 @@ public class BatchClient {
         public BatchClient(ApiKeyCredential credential, OpenAIClientOptions options = null);
         protected internal BatchClient(ClientPipeline pipeline, Uri endpoint, OpenAIClientOptions options);
         public virtual ClientPipeline Pipeline { get; }
+        public virtual BatchOperation CreateBatch(ReturnWhen returnWhen, BinaryContent content, RequestOptions options = null);
+        public virtual Task<BatchOperation> CreateBatchAsync(ReturnWhen returnWhen, BinaryContent content, RequestOptions options = null);
+        public virtual ClientResult GetBatches(string after, int? limit, RequestOptions options);
+        public virtual Task<ClientResult> GetBatchesAsync(string after, int? limit, RequestOptions options);
+    }
+    public class BatchOperation : OperationResult {
+        public override bool IsCompleted { get; protected set; }
+        public override ContinuationToken? RehydrationToken { get; protected set; }
```

Comment:
The SCM type is called `ContinuationToken` because it is used for both PageCollection rehydration/continuation and LRO rehydration.  IIRC we had discussed this previously, and you had felt `ContinuationToken` would work for both, but if you are suggesting we rename the property to `ContinuationToken`, I am happy to make that change.  Changing the SCM type name I think would be harder.